### PR TITLE
xilinx/vivado: add xilinx-vivado metapackage

### PR DESCRIPTION
**Pull request by kowalewskijan**
Package checks:
- If Vivado is in PATH
- If Vivado version matches the metapackage version

### CODE REVIEW

File: xilinx/vivado/pre-link.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+LOCAL_VIVADO_VERSION=`vivado -version | sed 's|.*Vivado v\([0-9\.]*\).*|\1|' | sed '1!d'`
+
+if [[ -z $LOCAL_VIVADO_VERSION ]]; then
+	echo "Please add Vivado to PATH by sourcing settings64.sh, then install package."
```
Comment (mithro) [inline xilinx/vivado/pre-link.sh:6]: We don't want people to source `settings64.sh` as it mucks up the environment.

Comment (kowalewskijan) [reply xilinx/vivado/pre-link.sh]: I corrected message to `echo "Please add Vivado to PATH then install package."`

Comment (mithro) [reply xilinx/vivado/pre-link.sh]: Can we just check for common vivado install locations?

Comment (kowalewskijan) [reply xilinx/vivado/pre-link.sh]: Ok, I think we can look for vivado in `PATH` first and then if it is not present we can look in common installation path: `/opt/Xilinx/Vivado/<version>/bin/vivado`

Comment (mithro) [reply xilinx/vivado/pre-link.sh]: SGTM!

Comment (kowalewskijan) [reply xilinx/vivado/pre-link.sh]: Fixed

**Comment by mithro** [comment]
LGTM. Merge if green.

**Pull request merged**